Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -105,6 +125,30 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
             LABELS REQUIRES_tiaf;TIAF_shard_fixture
         )
 
+        ly_add_target(
+            NAME AzFramework.NativeUI.Tests ${PAL_TRAIT_TEST_TARGET_TYPE}
+            NAMESPACE AZ
+            FILES_CMAKE
+                Tests/framework_nativeui_tests_files.cmake
+            INCLUDE_DIRECTORIES
+                PRIVATE
+                    Tests
+                    ${test_pal_dir}
+            BUILD_DEPENDENCIES
+                PRIVATE
+                    AZ::AzFramework
+                    AZ::AzTest
+                    AZ::AzTestShared
+                    AZ::AzFrameworkTestShared
+            RUNTIME_DEPENDENCIES
+                AZ::ProcessLaunchTest
+        )
+        ly_add_googletest(
+            NAME AZ::AzFramework.NativeUI.Tests
+            LABELS REQUIRES_tiaf;TIAF_shard_fixture
+        )
```

Comment:
Yeah we can skip the TIAFs labels for the NativeUI test to avoid future problems